charts/flyte-binary/values.yaml

```diff
@@ -234,12 +234,10 @@ deployment:
   # lifecycleHooks Specify hooks to run in Flyte container before or after startup
   lifecycleHooks: {}
   # resources Resource limits and requests for Flyte container
-  # Uncomment and update to specify resources for deployment
-  # resources:
-  #   limits:
-  #     memory: 1Gi
-  #   requests:
-  #     cpu: 1
+  resources:
+    requests:
+      cpu: "4"
+      memory: "4G"
```

Comment:
nit: historically we've always used Binary units instead of Decimal. I know it's a matter of preference, but let's stick to one in our docs/examples/values files.